src/js/wizards/Buy/0_Trade/components/OfferTrade.jsx

```diff
@@ -105,38 +109,56 @@ class OfferTrade extends Component {
                      data-minvalue={0}
                      validations={[isNumber, lowerEqThan, higherThan]}
                      id="fiat-quantity-input"
-                     placeholder="Fiat quantity" onChange={(e) => onCurrencyChange(e.target.value)} step="any"/>
+                     placeholder={t('buyer.offerTrade.fiatQuantity')} onChange={(e) => onCurrencyChange(e.target.value)} step="any"/>
               <span className="input-icon mr-3">{currency.id}</span>
             </Col>
           </Row>
         </FormGroup>
         { limitless && <p className="mt-3 limits">
-        Limits: {limitDecimals(minToken)} {asset} to <span id="max-token">{limitDecimals(maxToken)} {asset}</span>
+          <Trans 
+            i18nKey="buyer.offerTrade.limits" values={{
+            min: limitDecimals(minToken),
+            max: limitDecimals(maxToken),
+            symbol: asset
+          }}>
+          Limits: 0 Token to <span id="max-token">N Token</span>
```

Comment:
You forgot to add the values 
```suggestion
          Limits: {{min}} {{symbol}} to <span id="max-token">{{max}} {{symbol}}</span>
```